js/lesson4/tutorial.md

```diff
@@ -227,214 +228,183 @@ Well done, you've finished! For a bonus, switch your `getGithubInfo` method to r
 
 > Coach... explain the difference between synchronous and asynchronous requests. There's a good explanation on [Mozilla Developer Network (MDN)](https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Synchronous_and_Asynchronous_Requests)
 
+## Exercise 2 - Trivia!
 
-## ~~Exercise 2 - BBC's tomorrow's TV schedule~~
 
-![](https://cdn0.iconfinder.com/data/icons/shift-free/32/Error-128.png)
-
-**Part 2 of this exercise is no longer possible as the API that it uses has been taken down by the BBC.**  
-**Sorry, we're working on fixing the exercise!**
-
-[Download](https://gist.github.com/despo/05cab2f0b38bc02318e7/download) the exercise files or clone them directly from github `git clone https://gist.github.com/05cab2f0b38bc02318e7.git`
-
-For the second exercise, we will build an application that retrieves tomorrow's TV schedule for each genre using BBC's API.
+For this exercise, we will build a web app that generates trivia questions, using this open trivia questions api: https://opentdb.com/api_config.php.
+[Download](https://gist.github.com/kyorkston/a2768287fe15fc693da3dc62e8a8d697/download) the exercise files or clone them directly from github `git clone https://gist.github.com/a2768287fe15fc693da3dc62e8a8d697.git`. Shoutout to [despo](https://gist.github.com/despo) for originially making these files!
 
 ### What we will be doing:
 
-1. Retrieve and render available genres using `http://www.bbc.co.uk/tv/programmes/genres.json`
+1. Retrieve and log quiz questions usinf the quiz API: `https://opentdb.com/api.php?amount=10`
```

Comment:
```suggestion
1. Retrieve and log quiz questions using the quiz API: `https://opentdb.com/api.php?amount=10`
```
I think we have a little typo

Reply:
Thanks for taking the time to update this tutorial 👍

Reply:
Thanks for having a look 😄